### PR TITLE
Delete everything about latest version of SDK.

### PR DESCRIPTION
**Pull request by chenrujun**
## Purpose

Now **main** branch depends on latest stable version of Spring Cloud Azure libraries and SDK libraries. So, everything about developing version of SDK should be deleted. These items should be updated:
1. Delete the dependency about developing version of SDK in the pom.xml.
2. Delete logic about building latest version of SDK in GitHub actions.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Comment by chenrujun** [comment]
Hi, @fangjian0423 please help to review this PR when you have time.

**Comment by stliu** [comment]
can you please provide context of this change,  instead of describe the action that this change performs?

focus on what and why.

File: pom.xml [hunk 1]
```diff
@@ -136,162 +136,6 @@
   </build>
 
   <profiles>
-    <profile>
-      <id>latest-sdk</id>
```
Comment (fangjian0423) [inline pom.xml:140]: could you tell me where use `latest-sdk` profile, didn't find it. thanks

Comment (chenrujun) [reply pom.xml:140]: Originally, it's used manually when running in command line like `mvn clean package -P latest-sdk`.

**Review by fangjian0423 — APPROVED** [review]
LGTM

**Pull request merged**